### PR TITLE
Update Australian Production Capacities

### PR DESCRIPTION
**Pull request by brandongalbraith**
Updates the production capacity source in the README for Australia, as well as /config/zones.json production capacity data for all Australian states except the Northern Territory.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -202,7 +202,7 @@ Production capacities are centralized in the [zones.json](https://github.com/tmr
    - Biomass, Hydro, Solar, Wind: [IRENA](https://www.irena.org/-/media/Files/IRENA/Agency/Publication/2020/Mar/IRENA_RE_Capacity_Statistics_2020.pdf)
    - Nuclear, Gas: [wikipedia.org](https://en.wikipedia.org/wiki/List_of_power_stations_in_Armenia)
 - Aruba: [WEB Aruba](https://www.webaruba.com/energy-production/power-production-figures)
-- Australia [wikipedia.org](https://en.wikipedia.org/wiki/Wind_power_in_Australia#Wind_power_by_state)
+- Australia [OpenNEM](https://opennem.org.au/facilities/all/)
```
Comment (q--) [inline README.md:205]: Given that Western Australia isn't in the NEM, shouldn't you keep the Wikipedia link for that one?

**Comment by brandongalbraith** [comment]
OpenNEM contains generation data for Western Australia even though it’s not
part of NEM.

**Comment by brandongalbraith** [comment]
https://opennem.org.au/facilities/wa1/

**Comment by q--** [comment]
Should've checked, my bad

**Comment by brandongalbraith** [comment]
No worries at all, until I went digging in OpenNEM I thought the same!

**Review by FelixDQ — APPROVED** [review]
Awesome! Thank you @brandongalbraith!

**Comment by brandongalbraith** [comment]
@FelixDQ No problem! Happy to help. It seems like that link I provided to OpenNEM generation capacity for Western Australia is hanging now when trying to load, but I did archive their entire Australian generation mix page into the Internet Archive and emailed OpenNEM to find out what's up with that page not loading properly currently.

**Comment by FelixDQ** [comment]
Alright, thank you. Hopefully, it comes online again :-)

If it doesn't then it might be worth changing the url to the internet archive.

**Pull request merged**